docs/en/rst/installing/db_server.rst

```diff
@@ -3,16 +3,13 @@
 Database Server
 ###############
 
-Bugzilla requires a database to store its data. We recommend either MySQL or
-PostgreSQL for production installations. Oracle 10 should work fine, but very
-little or no testing has been done with Oracle 11 and 12. SQLite is easy to
-configure but, due to its limitations, it should only be used for testing
-purposes and very small installations.
+Bugzilla requires a database to store its data. We recommend either MariaDB, MySQL or PostgreSQL for production installations. Oracle 10 should work fine, but very little or no testing has been done with Oracle 11 and 12. SQLite is easy to configure but, due to its limitations, it should only be used for testing purposes and very small installations.
 
 .. toctree::
    :maxdepth: 1
 
    mysql
+   mariadb
```

Comment:
This would expect a mariadb.rst to exist.

So should we be creating this file and only minimally be editing mysql.rst?

Reply:
Agreed. Treat them as two fully separate entities now. Clone the existing mysql file for mariadb then edit it to be appropriate for Maria.